java/pgv-java-validation/pom.xml

```diff
@@ -21,18 +21,41 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
This is a required compile dependency - updated to pull it in explicitly and also enabled dependency analysis to catch any additional required dependencies which aren't declared.